web/l10n_utils.js

```diff
@@ -81,6 +81,7 @@ const DEFAULT_L10N_STRINGS = {
   printing_not_ready: "Warning: The PDF is not fully loaded for printing.",
   web_fonts_disabled:
     "Web fonts are disabled: unable to use embedded PDF fonts.",
+  freetext_default_content: "Enter some text...",
```

Comment:
This should use proper ellipsis instead:
```suggestion
  freetext_default_content: "Enter some text…",
```